config.js

```diff
@@ -247,6 +247,8 @@ var config = {
     // maintenance at 01:00 AM GMT,
     // noticeMessage: '',
 
+    // Disables the speech to text transcription subtitles panel.
+    disableTranscriptionSubtitles: false,
```

Comment:
I get the feeling this might be better off in interfaceConfig than config. What exactly the difference is between the two I can't say other than interfaceConfig was made as a way to toggle styling but eventually got bloated to include making UI elements display or not display.